common/constants/shared.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-import CSS from 'csstype';
+/* eslint-disable no-unused-vars */
```

Comment:
Can you please remove all the es-lint disabled lines. I see 24 occurrences of explicit disablement added in the PR.

Reply:
got it, removed